### PR TITLE
Fix scheme4

### PR DESCRIPTION
**Pull request by liwt31**
In scheme 4 all electron DOFs are in the same site.

Previously there's no zero exciton space in scheme4, which leads to a number of inconveniences.

This PR:
- add more docs (comments) on the scheme4 MPO
- add zero exciton state and QN for scheme4. All workarounds such as  `VirtualOnSite` can be removed

### CODE REVIEW

File: renormalizer/model/mlist.py [hunk 1]
```diff
@@ -43,7 +43,7 @@ def __init__(self, mol_list: List[Mol], j_matrix: Union[Quantity, np.ndarray, No
             for imol, mol in enumerate(mol_list):
                 if imol == len(mol_list) // 2:
                     self._e_idx = [len(self.pbond_list)] * len(mol_list)
-                    self.pbond_list.append(len(mol_list))
+                    self.pbond_list.append(len(mol_list)+1)
```
Comment (jjren) [inline renormalizer/model/mlist.py]: What is this +1?

**Comment by jjren** [comment]
@liwt31  Actually, I can't follow each line of the code now. Please recheck it by yourself. If you think this PR is ok, just merge it.
I will spend more time to get familiar with the code.

File: renormalizer/model/mlist.py (hunk 1 above)
Comment (liwt31) [reply renormalizer/model/mlist.py]: The physical space now contains zero exciton space, the basis are: |vacuum>, |1>...|m>

**Comment by liwt31** [comment]
It's alright. I think it's okay to let this PR stay around for a while (a month maybe?)

**Comment by jjren** [comment]
> It's alright. I think it's okay to let this PR stay around for a while (a month maybe?)

Ok.

**Comment by liwt31** [comment]
I'll merge this one soon because it seems OK to me...

**Comment by jjren** [comment]
> I'll merge this one soon because it seems OK to me...

No problem.

**Pull request merged**